app/brave_main_delegate.cc

```diff
@@ -165,6 +179,12 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
   command_line.AppendSwitch(switches::kEnableDomDistiller);
   command_line.AppendSwitch(switches::kNoPings);
 
+  auto update_url = GetUpdateURLHost();
```

Comment:
I'd rather use std::string instead of auto, because it's not clear whether this is GURL or string

Reply:
resolved!